partitura/score.py

```diff
@@ -2990,8 +3001,16 @@ class Score(object):
         the identifier should not start with a number.
     partlist : `Part`, `PartGroup` or list of `Part` or `PartGroup` instances.
         List of  `Part` or `PartGroup` objects.
-    title: str, optional
-        Title of the score.
+    work_title: str, optional
+        Work title of the score, if applicable.
+    work_number: str, optional
+        Work number of the score, if applicable.
```

Comment:
Maybe add a test to see if the extra information is parsed correctly, something simple for an existing score like:
`self.assert_true(score.work_title = "some work title")`